### PR TITLE
F scoring

### PR DESCRIPTION
**Pull request by wpoynter**

### CODE REVIEW

File: src/main/event/ScoreChangedEvent.hpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#ifndef SCORE_CHANGED_EVENT_HPP
+#define	SCORE_CHANGED_EVENT_HPP
+
+#include "Event.hpp"
+
+namespace gs {
+
+/**
+ * Event representing the destruction of a player.
```
Comment (smokyturbo) [inline src/main/event/ScoreChangedEvent.hpp]: Copy n' paste fail!

File: src/main/event/ScoreChangedEvent.hpp [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ */
+class ScoreChangedEvent : public Event {
+public:
+	/**
+	 * Constructor
+	 * @param _entityId Entity id
+	 */
```
Comment (smokyturbo) [inline src/main/event/ScoreChangedEvent.hpp]: ^

File: src/main/event/ScoreChangedEvent.hpp [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	 * @return Event type
+	 */
+	virtual EventEnum getType() const {
+		return SCORE_CHANGED_EVENT;
+	}
+
+	/**
+	 * Get the entity id
+	 * @return Entity id
+	 */
```
Comment (smokyturbo) [inline src/main/event/ScoreChangedEvent.hpp]: ^

File: src/main/event/ScoreChangedEvent.hpp [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	int getScore() const {
+		return score;
+	}
+
+private:
+	/** Screen score */
```
Comment (smokyturbo) [inline src/main/event/ScoreChangedEvent.hpp]: Screen score? The new score?

**Comment by wpoynter** [comment]
resolves #64

File: src/main/event/ScoreChangedEvent.hpp (hunk 1 above)
Comment (jademonkey) [inline src/main/event/ScoreChangedEvent.hpp]: Also do we need the indentation one line 2?

**Pull request merged**